pyigm/cgm/analysis.py

```diff
@@ -53,11 +64,18 @@ def dndx_rvir(Lrng=(0.001, 10), beta=0.2, rvir_Lstar=250.*u.kpc,
     phi_str_cgs = (phi_str_pref * 1e-2 * hubb**3) / u.Mpc**3
```

Comment:
Line 42 or 47 should contain a note that the phi_str needed is actually Phi* times 100.